apps/accounts/views/oauth2_profile.py

```diff
@@ -36,17 +35,6 @@ def openidconnect_userinfo(request):
     return JsonResponse(data)
 
 
-@require_GET
-@login_required()
-def userinfo_w_login(request):
-    """
-    OIDC Style userinfo
-    """
-    user = request.user
```

Comment:
This seems to be a way to get the same information as `openidconnect_userinfo` but with a different authentication path. Since there's a different auth path I think it makes sense to leave it as a seprate function.